### PR TITLE
Update Valentine's Day Origin dialogue

### PR DESCRIPTION
**Pull request by landofrhymeandraisins**
Valentine's Day origin edit, as discussed [here.](https://github.com/Monika-After-Story/MonikaModDev/issues/5340)

### CODE REVIEW

**Comment by MisterSimple** [comment]
You based this PR to the `master` branch, can you please change that so that it's based to the `content` branch instead?

File: Monika After Story/game/script-holidays.rpy [hunk 1]
```diff
@@ -5562,20 +5562,20 @@ init 5 python:
...
-    m 3euc "Before his death, he sent a love letter to her signed with 'From your Valentine.'"
-    m 1dsc "He was executed on February 14, 269 AD."
-    m 3eua "Such a noble cause, don't you think?"
+    m 1euc "The legends vary, but it dates back to the third century in Rome, when Christians were still persecuted by the Roman government."
+    m 3eud "Around this time, Emperor Claudius II had forbidden Christians from marrying, which a clergyman named Valentine decided was unfair."
+    m 3rsc "Against the orders of the emperor, he married Christians in secret."
+    m 3esc "Another version of the story is that Roman soldiers weren't allowed to be married, so by marrying people Valentine was saving them from conscription into the army."
+    m 1dsd "Either way, Valentine was caught and sentenced to death."
+    m 1euc "While in jail, he befriended the jailer's daughter and cured her blindness. Some say he even fell in love with her."
+    m 3euc "Unfortunately, this wasn't enough to save him. But before he died, he sent a letter to her, which he signed, "Your Valentine.""
```
Comment (multimokia) [inline Monika After Story/game/script-holidays.rpy]: Consistency, should use single quotes within these. Additionally, double quotes would need to be escaped (`\"`") to work.
```suggestion
    m 3euc "Unfortunately, this wasn't enough to save him. But before he died, he sent a letter to her, which he signed, 'Your Valentine.'"
```

**Comment by landofrhymeandraisins** [comment]
> 
> 
> You based this PR to the `master` branch, can you please change that so that it's based to the `content` branch instead?

I'm fairly new to github, could you tell me how to do that? Thanks!

**Comment by multimokia** [comment]
See [this entry on our FAQ](https://github.com/Monika-After-Story/MonikaModDev/wiki/FAQ#i-put-my-pull-request-in-the-wrong-branch-and-dont-want-to-suffer-the-wrath-of-devs-how-do-i-fix-it)

File: Monika After Story/game/script-holidays.rpy [hunk 2]
```diff
@@ -5562,20 +5562,20 @@ init 5 python:
...
-    m 3eua "Such a noble cause, don't you think?"
+    m 1euc "The legends vary, but it dates back to the third century in Rome, when Christians were still persecuted by the Roman government."
+    m 3eud "Around this time, Emperor Claudius II had forbidden Christians from marrying, which a clergyman named Valentine decided was unfair."
+    m 3rsc "Against the orders of the emperor, he married Christians in secret."
+    m 3esc "Another version of the story is that Roman soldiers weren't allowed to be married, so by marrying people Valentine was saving them from conscription into the army."
+    m 1dsd "Either way, Valentine was caught and sentenced to death."
+    m 1euc "While in jail, he befriended the jailer's daughter and cured her blindness. Some say he even fell in love with her."
+    m 3euc "Unfortunately, this wasn't enough to save him. But before he died, he sent a letter to her, which he signed, 'Your Valentine.'"
+    m 1dsc "He was executed on February 14, 269 AD, and later canonized as a saint."
+    m 3eua "To this day, it's still traditional to sign love letters with "Your Valentine!""
```
Comment (multimokia) [inline Monika After Story/game/script-holidays.rpy]: One more
```suggestion
    m 3eua "To this day, it's still traditional to sign love letters with 'Your Valentine!'"
```

File: Monika After Story/game/script-holidays.rpy [hunk 3]
```diff
@@ -5562,20 +5562,20 @@ init 5 python:
...
-    m 3eua "Such a noble cause, don't you think?"
+    m 1euc "The legends vary, but it dates back to the third century in Rome, when Christians were still persecuted by the Roman government."
+    m 3eud "Around this time, Emperor Claudius II had forbidden Christians from marrying, which a clergyman named Valentine decided was unfair."
+    m 3rsc "Against the orders of the emperor, he married Christians in secret."
+    m 3esc "Another version of the story is that Roman soldiers weren't allowed to be married, so by marrying people Valentine was saving them from conscription into the army."
+    m 1dsd "Either way, Valentine was caught and sentenced to death."
+    m 1euc "While in jail, he befriended the jailer's daughter and cured her blindness. Some say he even fell in love with her."
+    m 3euc "Unfortunately, this wasn't enough to save him. But before he died, he sent a letter to her, which he signed, 'Your Valentine.'"
+    m 1dsc "He was executed on February 14, 269 AD, and later canonized as a saint."
+    m 3eua "To this day, it's still traditional to sign love letters with 'Your Valentine!'"
```
Comment (MisterSimple) [inline Monika After Story/game/script-holidays.rpy]:
```suggestion
    m 3eua "To this day, it's still traditional to sign love letters with 'Your Valentine'!"
```

**Pull request merged**